apps/docs/.vitepress/config.mts

```diff
@@ -127,15 +151,16 @@ export default defineConfig({
             { icon: "twitter", link: "https://twitter.com/SunodoProject" },
             { icon: "github", link: "https://github.com/sunodo/sunodo" },
         ],
-        /*
-        TODO: request algolia search integration
         search: {
             provider: "algolia",
             options: {
-                appId: "0000000000",
-                apiKey: "xxxxxxx",
+                appId: "A3KGKST4NL",
+                apiKey: "4808497d4d3970d37a6c75f58ee1fccf",
```

Comment:
Is it safe to be public?

Reply:
Yes, it is safe.

Reply:
See for example [this](https://github.com/wagmi-dev/abitype/blob/bb4bf39ec242dc921f87f633eba0ffb54cc805e0/docs/.vitepress/config.ts#L61)